src/auth.js

```diff
@@ -47,6 +47,11 @@ function * signInWithRedirect (authProvider) {
   yield call([auth, auth.signInWithRedirect], authProvider)
 }
 
+function * signUpWithEmailAndPassword (email, password) {
+  const auth = this.app.auth()
+  return yield call([auth, auth.signUpWithEmailAndPassword], email, password)
```

Comment:
I can't find `signUpWithEmailAndPassword` in the [docs](https://firebase.google.com/docs/reference/js/firebase.auth.Auth), did you mean to use [`createUserWithEmailAndPassword`](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#createUserWithEmailAndPassword) instead?

If so I think we should change the name of this saga as well to match the name of the firebase SDK method being called. :)

Reply:
oops, ok i fix it ;)